verification-annotations/lib/seahorn.c

```diff
@@ -0,0 +1,21 @@
+#include <stdlib.h>
+#include <stdint.h>
+
+void __VERIFIER_error() {}
```

Comment:
I would have expected this to call a function to indicate that an error occurred?